portality/settings.py

```diff
@@ -626,6 +626,19 @@
     }
 }
 
+# Extension Map from dataobj coercion declarations to ES mappings.
+# This is useful when some extensions required for some objects additional to defaults.
+# ~~->DataObj:Library~~
+# ~~->Seamless:Library~~
+DATAOBJ_TO_MAPPING_EXTENSIONS = {
```

Comment:
Can we give this a more descriptive name like `DATAOBJ_TO_MAPPING_COPY_TO_EXTENSIONS`

Reply:
Updated